resources/user-agents/apps/facebook/facebook-app.json

```diff
@@ -195,22 +195,21 @@
             "N9519": "ZTE N9519",
             "Nexus 5": "LG Nexus 5",
             "R1 HD": "BLU R1 HD",
-            "SAMSUNG-SM-G891A": "Samsung SM-G891A",
```

Comment:
If you look into line 213 of this file, there was the same device without the Samsung-Prefix. I think, its easier to handle without that prefix.